client/fvnsMcastAddrBlk_service.go

```diff
@@ -0,0 +1,48 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/models"
+)
+
+func (sm *ServiceManager) CreateAbstractionofIPAddressBlock(to string, _from string, multicast_address_pool string, description string, fvnsMcastAddrBlkAttr models.AbstractionofIPAddressBlockAttributes) (*models.AbstractionofIPAddressBlock, error) {
```

Comment:
CreateAbstractionofIPAddressBlock -> CreateMulticastAddressBlock 

Also adapt all other instance to similar format.

Reply:
done